ezstaking/chains.json

```diff
@@ -334,6 +334,15 @@
         "run_time": "every 5 minutes",
         "minimum_reward": 100000
       }
+    },
+    {
+      "name": "kyve",
+      "address": "stridevaloper1gjtvly9lel6zskvwtvlg5vhwpu9c9wawm6kh6u",
```

Comment:
Wrong validator address

Reply:
Indeed... thank you very much @tombeynon